ores/ores.py

```diff
@@ -30,7 +28,7 @@ def main():
         sys.stderr.write(USAGE)
         sys.exit(1)
     elif sys.argv[1] in ("-h", "--help"):
-        sys.stderr.write(__doc__ + "\n")
+        sys.stderr.write(__doc__ % { "usage": USAGE })
```

Comment:
Makes it a bit harder to read, but I appreciate that it reduces code duplication.  I wish there was a better way to do this.